traffic_monitor/todata/todata.go

```diff
@@ -162,44 +154,71 @@ func (d TODataThreadsafe) Update(to towrap.TrafficOpsSessionThreadsafe, cdn stri
 	json := jsoniter.ConfigFastest
 	err = json.Unmarshal(crConfigBytes, &crConfig)
 	if err != nil {
-		return fmt.Errorf("Error unmarshalling CRconfig: %v", err)
+		return fmt.Errorf("unmarshalling CRconfig: %v", err)
 	}
 
-	newTOData.DeliveryServiceServers, newTOData.ServerDeliveryServices, err = getDeliveryServiceServers(crConfig)
-	if err != nil {
-		return err
-	}
+	// TODO: remove the fallback behavior on the CRConfig in ATC 8.0 (https://github.com/apache/trafficcontrol/issues/6627)
+	newTOData.DeliveryServiceServers, newTOData.ServerDeliveryServices = getDeliveryServiceServers(crConfig, mc)
 
-	newTOData.DeliveryServiceTypes, err = getDeliveryServiceTypes(crConfig)
+	// TODO: remove the fallback behavior on the CRConfig in ATC 8.0 (https://github.com/apache/trafficcontrol/issues/6627)
+	newTOData.DeliveryServiceTypes, err = getDeliveryServiceTypes(crConfig, mc)
 	if err != nil {
-		return fmt.Errorf("Error getting delivery service types from Traffic Ops: %v\n", err)
+		return fmt.Errorf("getting delivery service types from Traffic Ops: %v\n", err)
```

Comment:
Can you remove the newlines in these, too? In TO I'd also suggest error-wrapping, but that can be very slow and TM needs all the performance it can get.